docs/DatabaseCatalogAPI.yaml

```diff
@@ -233,13 +233,24 @@ paths:
 
 components:
   schemas:
+    Redirect:
+      type: object
+      properties:
+        target_id:
+          description: ID of the feed to which the current feed is being redirected.
```

Comment:
Minor edit of description for clarity: `The feed ID that should be used in replacement of the current one.`